### PR TITLE
feat: ensure firefox (flatpak) has hardware decoding turned on by default

### LINKED ISSUE #1730: Ensure firefox has hardware decoding turned on
Did a new reinstall on a device and I had to manually set hw accel on in Firefox: 

Bazzite has this: https://github.com/ublue-os/bazzite/blob/main/system_files/desktop/shared/usr/share/ublue-os/firefox-config/01-bazzite-global.js

I could have sworn we fixed this ages ago and was surprised when the laptop turned into a jet engine.

### PR DESCRIPTION
**Pull request by tulilirockz**
Got this straight from Bazzite, just renamed a few thingies to make it Bluefin-themed :p - Fixes #1730

### CODE REVIEW

**Comment by bsherman** [comment]
I'm not sure about all these settings... I don't think they are required to enable hardware accelerated video decoding.

I don't have any of them and hw accel decode works well for me. (I happen to have been testing this topic lately).

File: system_files/silverblue/usr/share/ublue-os/firefox-config/03-bluefin-gnome.js
Comment (bsherman) [inline system_files/silverblue/usr/share/ublue-os/firefox-config/03-bluefin-gnome.js:1]: Let's skip this file for now as I'd like to do small, targeted changes and at the least, the rounded corner setting has nothing to do with hw accerlation.

**Review by bsherman — CHANGES_REQUESTED** [review]
I think we can go with this if it's only the `01-bluefin-global.js` for now.

I've been testing hardware acceleration a lot this past week. So while I do want us to default enable `media.ffmpeg.vaapi.enabled` and `gfx.webrender.all` is also probably fine, I'm hesitant on the other changes just yet.

File: system_files/silverblue/usr/share/ublue-os/firefox-config/03-bluefin-gnome.js
Comment (tulilirockz) [reply system_files/silverblue/usr/share/ublue-os/firefox-config/03-bluefin-gnome.js:1]: True! Lemme just fix this thing then

**Review by bsherman — APPROVED** [review]
Thank you!

**Pull request merged**